### DISCUSSION TITLE
Files not showing up in mounted S3 bucket after upgrading. (Windows)

### DISCUSSION THREAD

**Discussion by Hbalduf2121** [iterate-ch/cyberduck#14627] [Q&A] [answered]
We just had some users update from version 2.7.2 (9873) to version 4.13.8 (21236). These users are accessing a large S3 bucket with 
a lot of image files and following the update, none of the image files show up anymore. All the folders are intact and in the correct place but opening a folder that should be filled with image files either shows up empty or is stuck with "working on it" forever. 

Using version 2.7.2 I can still access the image files but not with the new version. It seems like a sync issue and I am unfamiliar with the sync feature, but sometime when entering a folder that should have files but appears empty the folder is deleted by mountain duck, and then repopulated, but with no images still. 

Strangely I have another S3 bucket with similar images that they are showing up in.

A repeated error I see in my log file is: 
```
[fs-1] ERROR ch.iterate.mountainduck.sync.lock.ReentrantPathLock - Missing lock for file Path{path='**ACTUAL PATH REDACTED**', type=[directory, placeholder]}
```

Thanks for any help!

**Comment by AliveDevil** [comment, marked as answer]
Service Desk response:

With 3.2.3 Mountain Duck hides items with storage class "GLACIER".
The items in question are all in GLACIER-class, thus weren't visible.

**Reply by dkocher** [reply]
Documented in [Restore from Glacier](https://docs.cyberduck.io/protocols/s3/#restore-from-glacier).
